modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php

```diff
@@ -185,9 +185,11 @@ public function __construct(
    * @aliases acr
    * @usage acms:config-reset
    *   Reset the configuration to the default.
-   * @usage acms:config-reset acquia_cms_article acquia_cms_common --scope=all
+   * @usage acms:config-reset acquia_cms_article acquia_cms_article acquia_cms_person --scope=all
```

Comment:
@chandan-singh7929 Argument: _acquia_cms_article_ is added twice. Possibly we need to remove one. Ex:
```suggestion
   * @usage acms:config-reset acquia_cms_article acquia_cms_person --scope=all
```

Reply:
@vishalkhode1 - Thanks, updated.